service-worker-registration/index.js

```diff
@@ -2,9 +2,14 @@ export const PROJECT_REVISION = '{{PROJECT_REVISION}}';
 
 let SUCCESS_HANDLERS = [];
 let ERROR_HANDLERS = [];
+const SCOPE = '{{SERVICE_WORKER_SCOPE}}';
```

Comment:
Instead of doing all the work here in the registration file itself, what if we just passed in the adjusted scope? Would make testing easier as well I think.
For example, in `_rollupTree` from the `ServiceWorkerBuilder`
```
SERVICE_WORKER_SCOPE: this.options.serviceWorkerScope || this.options.rootURL
```

Then your test file only needs to verify that the scope is passed in scope or the default scope.